src/lib/shell.html

```diff
@@ -33,8 +33,8 @@
             </div>
         </div>
     </div>
-    <div class="bp" tabindex="0" data-testid="bp">
-        <div class="bp-content" tabindex="0" data-testid="bp-content">
+    <div class="bp" data-testid="bp" tabindex="-1">
```

Comment:
Do both of these need to be programmatically focusable? It looks like `BaseViewer` focuses on `.bp-content` when `autoFocus` is enabled.

Reply:
The `ContentPreview` Element auto-focuses on `.bp` and the Preview SDK auto-focuses on `.bp-content`.